### PR TITLE
CHANGELOG.md,doc/; adds basic documentation of the `migrate` and `run ansible` commands

### PR DESCRIPTION
**Pull request by mhrivnak**

### CODE REVIEW

File: doc/sdk-cli-reference.md [hunk 1]
```diff
@@ -265,6 +284,24 @@ Create deploy/crds/app_v1alpha1_appservice_crd.yaml
 Create deploy/crds/app_v1alpha1_appservice_cr.yaml
 ```
 
+## run
+
+### ansible
+
+Runs as an ansible operator process. This is intended to be used when running
+in a Pod inside a cluster.
```
Comment (hasbro17) [inline doc/sdk-cli-reference.md]: We should add a note that users should always use `operator-sdk up local` as opposed to this. And that this command is largely intended for the SDK's internal lifecycle purposes.

Both here and in the CHANGELOG.

File: doc/sdk-cli-reference.md [hunk 2]
```diff
@@ -166,6 +166,25 @@ $ operator-sdk olm-catalog gen-csv --csv-version 0.1.1
...
 ```
 
+## migrate
+
+Adds a main.go source file and any associated source files for an operator that
+is not of the "go" type.
+
+### Example
+
+```bash
```
Comment (hasbro17) [inline doc/sdk-cli-reference.md]:
```suggestion
```console
```

**Comment by hasbro17** [comment]
Can we also make the `--help` flag output for the `operator-sdk run` command more verbose or the same as the docs here. With the note pointing out the preference of `operator-sdk up local`.

https://github.com/operator-framework/operator-sdk/blob/master/commands/operator-sdk/cmd/run.go#L28

https://github.com/operator-framework/operator-sdk/blob/master/commands/operator-sdk/cmd/run/ansible.go#L30

**Comment by mhrivnak** [comment]
The CI failure is due to a new bug in the python client, unrelated to this PR. @fabianvf is working on a fix now.

**Pull request merged**